### PR TITLE
Fixed system user to be able to pull UID/GID and then added it to the…

### PR DESCRIPTION
**Pull request by jamesbrown7**
… ExportUserToCSV code

### CODE REVIEW

File: jcapi_test.go [hunk 1]
```diff
@@ -16,17 +16,17 @@ var testAPIKey string = os.Getenv("JUMPCLOUD_APIKEY")
...
+		FirstName:        "Test",
+		LastName:         "User",
+		Email:            "testuser@jumpcloud.com",
+		Password:         "test!@#$ADSF",
+		Activated:        true,
+		Sudo:             true,
+		Uid:              "2244",
+		Gid:              "2244",
+		EnableManagedUid: true,
+		//TagList:           make([]string, 0),
```
Comment (rmulley) [inline jcapi_test.go]: This comment can be removed

**Comment by kusold** [comment]
:+1:

File: jcapi-systemusers.go [hunk 2]
```diff
@@ -93,13 +100,19 @@ func getJCUserFieldsFromInterface(fields map[string]interface{}, user *JCUser) {
 		user.ExternalSourceType = fields["external_source_type"].(string)
 	}
 
-	// Currently returned as int, not string (though they are posted as string),
+	// Currently returned as float64, not string (though they are posted as string),
 	// defect #96322248...
 	if _, exists := fields["unix_uid"]; exists {
-		user.Uid = getStringOrNil(fields["unix_uid"])
+		floatVal, ok := fields["unix_uid"].(float64)
```
Comment (tyoes) [inline jcapi-systemusers.go]: purely cosmetic, but makes for tighter code:

if floatVal, ok := fields["unix_uid"].(float64); ok {
            user.Uid = strconv.FormatInt(int64(floatVal), 10)
        }

File: jcapi-systemusers.go [hunk 3]
```diff
@@ -93,13 +100,19 @@ func getJCUserFieldsFromInterface(fields map[string]interface{}, user *JCUser) {
...
+		if ok {
+			user.Uid = strconv.FormatInt(int64(floatVal), 10)
+		}
 	}
-	if _, exists := fields["unix_gid"]; exists {
-		user.Gid = getStringOrNil(fields["unix_gid"])
+	if _, exists := fields["unix_guid"]; exists {
+		floatVal, ok := fields["unix_guid"].(float64)
```
Comment (tyoes) [inline jcapi-systemusers.go]: same terseness comment

File: jcapi-systemusers.go (hunk 2 above)
Comment (rmulley) [inline jcapi-systemusers.go]: +1

**Comment by rmulley** [comment]
:+1:

**Comment by tyoes** [comment]
I see no tests at all. Missed checkin?

**Pull request merged**